files-community/Files
# Fix: Fixed can't drag multiple files from the tags area

**Resolved / Related Issues**
   Closes #11958 

**Validation**
How did you test these changes?
   1. Click on a tag in the sidebar
   2. Select at least two items
   3. Drag them around
